kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/VirtualKafkaClusterReconciler.java

```diff
@@ -184,17 +156,11 @@ private static String joiningMessages(
         return Stream.of(serviceMsg).flatMap(Function.identity()).collect(Collectors.joining("; "));
     }
 
-    private static boolean hasAnyResolvedRefsFalse(List<Condition> conditions) {
-        return conditions.stream()
-                .anyMatch(c -> Condition.Type.ResolvedRefs.equals(c.getType())
-                        && Condition.Status.FALSE.equals(c.getStatus()));
-    }
-
     @NonNull
-    private static <R extends CustomResource<?, ?>> Stream<String> refsMessage(
-                                                                               String prefix,
-                                                                               VirtualKafkaCluster cluster,
-                                                                               TreeSet<? extends LocalRef<R>> refs) {
+    private static Stream<String> refsMessage(
+                                              String prefix,
+                                              VirtualKafkaCluster cluster,
+                                              TreeSet<LocalRef<?>> refs) {
```

Comment:
Why do we collect to a tree set to then test if its an empty stream and if not get the stream back from the set. Why not just accept `Stream<LocalRef<?>>` and then do `refs.sorted()`

Reply:
replaced with Stream